docs/multi-cluster-service.md

```diff
@@ -0,0 +1,184 @@
+# Multi-cluster Service Discovery
+
+Users are able to **export** and **import** services between clusters with [Multi-cluster Service APIs](https://github.com/kubernetes-sigs/mcs-api).
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install karmada refer to [quick-start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/karmada-bootstrap.sh` script which is also used to run our E2E cases.
+
+### Member Cluster Network
+
+Ensure that at least two clusters has been added to karmada, and the container networks between member clusters are connected.
+
+- If you use the `hack/karmada-bootstrap.sh` script to deploy karmada. The deployed karmada has three member clusters, and the container networks of the `member1` and `member2` are connected.
```

Comment:
- If you use the `hack/karmada-bootstrap.sh` script to deploy Karmada, Karmada will have three member clusters, and the container networks of the `member1` and `member2` will be connected.